src/config/dynamic-theme-fixes.config

```diff
@@ -274,6 +274,15 @@ INVERT
 
 ================================
 
+access.ing.de
+
+CSS
+.tile__info {
+  background-color: #fff !important;
```

Comment:
4 Indents & the hard coded color is good as it could otherwise break the QR code implementation